api/resources/translations/messages-es.properties

```
@@ -162,7 +162,7 @@ No\ registrations\ found = Ninguna inscripción encontrada para este horario. In
 No\ report\ selected = Ningún informe seleccionado
 No\ reports\ found = Ningún informe encontrado
 No\ schedules\ found = Ningún horario encontrado
-No\ submission = 
+No\ submission = Sin presentación
```

```suggestion
No\ submission = No comparta los datos del impacto 
```
I didn't find the usage of this in any page. But looking at the translation in English `Do not share impact data`, I suggested something along the same line.